Lib/test/test_isinstance.py

```diff
@@ -251,6 +251,25 @@ def test_isinstance_recursion_limit(self):
         # blown
         self.assertRaises(RecursionError, blowstack, isinstance, '', str)
 
+    def test_issubclass_refcount_handling(self):
+        # bpo-39382: abstract_issubclass() didn't hold item reference while
+        # peeking in the bases tuple, in the single inheritance case.
+        class A:
+            @property
+            def __bases__(self):
+                return (int, )
+
+        class B:
+            def __init__(self):
+                self.x = 1
```

Comment:
To me the test crashes on unpatched code even without adding a dummy member. So it is not strictly necessary. If you decide to keep it, please add a short comment.

Reply:
Hmm. Which Python version were you testing with? I don't get crashes with 3.8/3.7 (on Arch Linux)

Anyway, that's a problem, because it might "pass" in the CI just because it doesn't crash (Well, my idea with the `assertEqual`, given the hypothesis of the "another object gets the same memory position", was to ascertain it's still the `A` object. But as with all memory errors, we can't trust it).

For starters I'll test it in the CI this way and see that it fails (w/o the code fix). I'll also add a comment.

Reply:
Okay, the test with `self.x = 1` crashes in the CI. The test without failed on Mac and Ubuntu, but passed on Windows.. ?

I'll leave it this way, and add a comment explaining (or just stating that I'm not sure about it, but this exhibits the bug)

Reply:
I tested with 3.7/3.8/master on Ubuntu.

Thank you for testing and adding a comment.